internal/it/serialization.go

```diff
@@ -25,5 +25,8 @@ import (
 func SerializationTester(t *testing.T, f func(t *testing.T, config hazelcast.Config, clusterID, mapName string)) {
 	ensureRemoteController(true)
 	mapName := NewUniqueObjectName("map")
+	config := defaultTestCluster.DefaultConfig()
+	// SSL is never used with serialization tests.
+	config.Cluster.Network.SSL.Enabled = false
 	f(t, defaultTestCluster.DefaultConfig(), defaultTestCluster.ClusterID, mapName)
```

Comment:
should not we use config here?

```suggestion
	f(t, config, defaultTestCluster.ClusterID, mapName)
```

Also, does making SSL enabled break some tests? If not, it may not be needed to be disabled.